app/src/main/java/com/hyperion/ths/marvel_03/data/source/HeroRepository.java

```diff
@@ -0,0 +1,41 @@
+package com.hyperion.ths.marvel_03.data.source;
+
+import com.hyperion.ths.marvel_03.data.model.Hero;
+import com.hyperion.ths.marvel_03.data.source.HeroDataSource;
```

Comment:
WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#ImportFromSamePackage